PostProcess/Bloom/src/Bloom.cpp

```diff
@@ -152,6 +152,8 @@ Int32 Bloom::ComputeMipCount(Uint32 Width, Uint32 Height, float Radius)
 
 void Bloom::ComputePrefilteredTexture(const RenderAttributes& RenderAttribs)
 {
+    auto const& SupportedFeatures = RenderAttribs.pPostFXContext->GetSupportedFeatures();
```

Comment:
This is only specific to bloom, no need to have this in FX context

Reply:
Let's keep it for consistency with other post effects. It's not a fact that it won't be needed in the future